db/fixtures/miq_shortcuts.yml

```diff
@@ -4,257 +4,453 @@
   :url: /dashboard/show
   :rbac_feature_name: dashboard_view
   :startup: true
+- :name: miq_report
+  :description: Cloud Intel / Reports
+  :url: /report/explorer
+  :rbac_feature_name: miq_report
+  :startup: true
 - :name: chargeback
-  :description: Cloud Intelligence / Chargeback
+  :description: Cloud Intel / Chargeback
   :url: /chargeback/explorer
   :rbac_feature_name: chargeback
   :startup: true
-- :name: report_explorer
-  :description: Cloud Intelligence / Reports
-  :url: /report/explorer
-  :rbac_feature_name: miq_report
-  :startup: true
 - :name: timelines
-  :description: Cloud Intelligence / Timelines
+  :description: Cloud Intel / Timelines
   :url: /dashboard/timeline
   :rbac_feature_name: timeline
   :startup: true
 - :name: rss
-  :description: Cloud Intelligence / RSS
-  :url: /alert
+  :description: Cloud Intel / RSS
+  :url: /alert/show_list
   :rbac_feature_name: rss
   :startup: true
-- :name: service_explorer
+- :name: consumption
+  :description: Consumption / Dashboard
+  :url: /consumption/show
+  :rbac_feature_name: consumption
+  :startup: true
+- :name: services
   :description: Services / My Services
   :url: /service/explorer
   :rbac_feature_name: service
   :startup: true
-- :name: catalog_explorer
+- :name: catalogs
   :description: Services / Catalogs
   :url: /catalog/explorer
   :rbac_feature_name: catalog
   :startup: true
-- :name: requests
+- :name: vm_explorer
+  :description: Services / Workloads
+  :url: /vm_or_template/explorer
+  :rbac_feature_name: vm_explorer
+  :startup: true
+- :name: miq_requests
   :description: Services / Requests
   :url: /miq_request/show_list
   :rbac_feature_name: miq_request_show_list
   :startup: true
-- :name: vms_myvms
-  :description: Services / Workloads / VMs & Instances
-  :url: /vm_or_template/explorer?accordion=vms_instances_filter
-  :rbac_feature_name: vms_instances_filter_accord
-  :startup: true
-- :name: vms_mytemplates
-  :description: Services / Workloads / Templates & Images
-  :url: /vm_or_template/explorer?accordion=templates_images_filter
```

Comment:
@GregP we need to keep all VM explorer related start page entries that allows user to be able to set a specific accordion as their start page. See my screenshot below where i have set "Services / Workloads/ Templates & Images" accordion to be my start page.
![screenshot from 2017-05-13 13-27-00](https://cloud.githubusercontent.com/assets/3450808/26027686/421b7b62-37e0-11e7-8e57-a4e6f4e380a8.png)